docs/device-preparation.md

```diff
@@ -32,6 +32,14 @@ Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Adva
 Make sure the destination `WKWebView` and/or `JSContext` component have [`isInspectable`](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable) property set to `true`.
 Read [Enabling the Inspection of Web Content in Apps](https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/) for more details on this property.
 
+#### Chrome v115+ and iOS 16.4+ support Web testing availability on Chrome
+
+As `isInspectable` is available, Chrome started providing the availability in the release package.
```

Comment:
Chrome browser for iOS now provides a remote debugging feature for release versions of the app after the `isInspectable` property has been introduced by Apple.